native/monogame/vulkan/MGG_Vulkan.cpp

```diff
@@ -4331,7 +4331,7 @@ static MGG_Buffer* MGVK_Buffer_Create(MGG_GraphicsDevice* device, MGBufferType t
 	return buffer;
 }
 
-MGG_Buffer* MGG_Buffer_Create(MGG_GraphicsDevice* device, MGBufferType type, mgint sizeInBytes)
+MGG_Buffer* MGG_Buffer_Create(MGG_GraphicsDevice* device, MGBufferType type, mgint sizeInBytes, const char* name)
```

Comment:
Note this is something we should be doing with most graphics resources as assigning names to them make it MUCH easier to debug using the different platform GPU debuggers.

The concern is extra garbage/performance hits from passing strings how we do at the moment.